grid/grid.py

```diff
@@ -44,6 +44,10 @@
     help="If this flag is used a SQLAlchemy DB URI is generated to use a local db.",
 )
 
+parser.add_argument(
+    "--id", dest="node_id", action="store_true", help="PyGrid Node ID",
+)
+
```

Comment:
```suggestion
parser.add_argument(
    "--id", 
    dest="node_id", 
    action="store_true", help="PyGrid Node ID",
)
```
Same update format here